pwiz_tools/Skyline/Model/Lib/SpectrumRanker.cs

```diff
@@ -701,6 +719,10 @@ private RankedMI CalculateRank(RankingState rankingState, RankedMI rankedMI)
                     continue;
                 }
 
+                //custom ions have been already matched above. No need to do anything.
```

Comment:
Your comment style is at odds with the project. Put a space after the // and capitalize the first letter.